commcare_connect/static/prelogin/contact-form.js

```
@@ -0,0 +1,20 @@
+(function () {
+  function mountForm() {
+    if (typeof hbspt === 'undefined' || !hbspt.forms) return false;
```

ESLint

[error] 'hbspt' is not defined.

(no-undef)